### PR TITLE
VEN-58 |  VEN-59 | Remove all traces of mock node server, use default env var

### PR DESCRIPTION
**Pull request by midori0507**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,4 +11,4 @@ COPY package.json package.json
 
 RUN yarn install --silent
 
-CMD ["yarn", "dev"]
\ No newline at end of file
+CMD ["yarn", "start"]
```
Comment (igordavydsson) [inline Dockerfile]: please, add new line

Comment (midori0507) [reply Dockerfile]: i dont get it :D

File: src/utils/api.ts [hunk 2]
```diff
@@ -1,13 +0,0 @@
-import axios from 'axios';
```
Comment (Nizar-Rahme) [inline src/utils/api.ts]: If it's no longer used in other places, please remove it from our dependencies.

File: Dockerfile (hunk 1 above)
Comment (igordavydsson) [reply Dockerfile]: new line at the end of the file I mean

Comment (igordavydsson) [reply Dockerfile]: newline

**Pull request merged**